config/copycat-standalone.properties

```diff
@@ -16,12 +16,21 @@
 # These are defaults. This file just demonstrates how to override some settings.
 bootstrap.servers=localhost:9092
 
+# The converters specify the format of data in Kafka and how to translate it into Copycat data. Every Copycat user will
+# need to configure these based on the format they want their data in when loaded from or stored into Kafka
 key.converter=org.apache.kafka.copycat.json.JsonConverter
```

Comment:
It's right below this -- both `key.converter.schemas.enable=true` and `value.converter.schemas.enable=true`.

This is a new configuration convention which is something we might want to discuss. This was the best way I could come up with to avoid the potential conflicts.